examples/run-app.sh

```diff
@@ -33,10 +35,13 @@ ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P
 
 # Get App attributes (Name and Port Number)
 APP_PORT=50008
-while getopts a:p:h flag
+export SDV_MIDDLEWARE_TYPE=dapr
+while getopts a:dnp:h flag
 do
     case "${flag}" in
         a) APP_NAME=${OPTARG};;
+        d) export SDV_MIDDLEWARE_TYPE=dapr;;
```

Comment:
Both flags are excluded each other. Probably flag with parameter shall be used, aka `-r $(OPTARG)`

Reply:
Yes, but then it gets lengthy. Did a proposal improving the help output.
Also, don't want to spend too much effort here for now

Reply:
This won't result in an error, since the later one is just replacing the other. So I would also leave it for now